<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWidget</class>
 <widget class="QWidget" name="RegistrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: #88000000;
}

QDialog, QDialog QLabel {
	background-color: #440000;
	color: red
}

QLineEdit {
	color: white;
	border: 2px solid gray;

	font-size: 18px;
	padding: 6px;

	border-top-left-radius: 20px;
	border-bottom-right-radius: 20px;
}

QLineEdit:hover {
	border-color: white;
}

QLineEdit:focus {
	border-color: yellow;
}

QLabel {
	font-size: 24px;
	color: rgb(255, 234, 0);

	border-top-left-radius: 30px;
	border-bottom-right-radius: 30px;

	padding: 6px;
}

QPushButton {
	color: yellow;
	font-size: 14px;
	padding: 6px 18px 6px 18px;

	border-top-left-radius: 24px;
	border-bottom-right-radius: 24px;

	border: 2px solid transparent;
}

QPushButton:hover {
	border-color: yellow;
}

QPushButton:pressed {
	background-color: yellow;
	color: #88000000;
}

QPushButton:disabled {
	background-color: #88888888;
	color: darkgray;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Регистрация</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="editPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Пароль (минимум 6 символов)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>29</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="editLogin">
     <property name="placeholderText">
      <string>Логин (минимум 6 символов)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="buttonRegistration">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="editRepeatPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Повторите пароль</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="editNickname">
     <property name="placeholderText">
      <string>Псевдоним (Минимум 6 символов)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
